README.md

```diff
@@ -31,13 +31,13 @@ myHex =
 Use helpers like `contrast` and `sufficientContrast` to verify that your font size, boldness, and colors meet accessibility standards.
 
 ```
```

Comment:
would you be open to a PR to add checks for this with elm-verify-examples?

Reply:
Yes!! (Although maybe after the major version release has actually occurred in order to not waste your time if this API doesn't make it 😅 )